api/src/db/db.service.ts

```diff
@@ -166,3 +177,16 @@ export class DbService {
         });
     }
 }
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
There's a reason for [this rule](https://typescript-eslint.io/rules/no-namespace/) 😉 

> TypeScript historically allowed a form of code organization called "custom modules" (module Example {}), later renamed to "namespaces" (namespace Example). Namespaces are an outdated way to organize TypeScript code. ES2015 module syntax is now preferred (import/export).

I don't think a namespace like this is needed, it's usually not how (internal) code is organized. Just put the type directly at the top of this file or move it to a separate types file. I think the first option is fine for now - other classes using the DbService don't necessarily need to import the type, as the editor/compiler will still understand which type it is when passing the argument as an object.